src/components/PeopleList.tsx

```diff
@@ -0,0 +1,115 @@
+// PeopleList.js
+import classNames from 'classnames';
+import { Link } from 'react-router-dom';
+import { Loader } from './Loader';
+
+type Person = {
+  name: string;
+  sex: string;
+  born: number;
+  died: number;
+  motherName: string | null;
+  fatherName: string | null;
+  slug: string;
+};
+
+type PeopleListProps = {
+  people: Person[];
+  isLoading: boolean;
+  highlightedPersonSlug?: string; // Новый проп для передачи выделенного slug
+};
+
+export const PeopleList = ({
+  people,
+  isLoading,
+  highlightedPersonSlug,
+}: PeopleListProps) => {
+  const getHref = (name) => {
+    return people.find(per => name === per.name);
```

Comment:
The `getHref` function is expected to return a string or URL, but it currently returns a `Person` object. This might cause confusion when used in the `to` prop of the `Link` component. Consider renaming the function to better reflect its purpose, such as `findPersonByName`, and ensure it returns the correct type.